### ISSUE TITLE
Background playback: (live) streaming stops.

### ISSUE THREAD

**Issue by WebworkrNet** [mediathekview/zapp#383]
Without the snooze function (#194) activated, playback occasionally stops after a while without resuming.

Related to question #241.

**Comment by WebworkrNet** [comment]
Active downloads and queued downloads during backround playback presumably prevent this unwanted behavior.

**Comment by WebworkrNet** [comment]
Recent observations have shown that the error also occurs when the sleep function is activated. I will adapt the original error description.

The ticket has received the label “Feedback needed”. I always send crash reports via email whenever possible. Is there anything else I can do?

**Comment by cemrich** [comment]
Does the streaming stop with a crash? If so, can you please point me to the specific crash (first line of the stack trace would be helpful)?

**Comment by WebworkrNet** [comment]
I'll have to pay careful attention to that next time.

As a rule, I send the report by email for every crash if the app asks me to do so, as mentioned before.

Can the energy settings also be relevant in this context, as in the similar #384?

**Comment by cemrich** [comment]
Yes, this absolutely could be related to the energy settings. Thank you very much for testing!

**Comment by WebworkrNet** [comment]
I want to emphasize that the statement in #384 about overheating is an assumption. I'll have to take a closer look at whether this can also be used as a possible justification here.
